README.md

```diff
@@ -47,3 +51,9 @@ cat <<EOF > server.json
 EOF
 pm2 start server.json
 ```
+
+## TESTING
```

Comment:
I'm going to have to vote for no all caps here haha :P

IT MIGHT SEEM LIKE WE'RE YELLING!! :D :D